### DISCUSSION TITLE
zh/front/base/concept

### DISCUSSION THREAD

**Discussion by giscus** [mineadmin/MineAdmin#352] [Q&A]
# zh/front/base/concept

企业级后台管理系统

http://localhost:5173/zh/front/base/concept.html

**Comment by 1286011129** [comment]
群答案是啥呀？进不去

**Comment by zds-s** [comment]
hyperf



---Original---
From: ***@***.***&gt;
Date: Thu, Jun 5, 2025 22:27 PM
To: ***@***.***&gt;;
Cc: ***@***.***&gt;;
Subject: Re: [mineadmin/MineAdmin] zh/front/base/concept (Discussion #352)




 
群答案是啥呀？进不去
